scripts/apidocs-gen/generate.sh

```diff
@@ -23,11 +23,23 @@ KUBERNETES_CONFIGURATION_CRDS_PACKAGE="github.com/kong/kubernetes-configuration"
 KUBERNETES_CONFIGURATION_CRDS_VERSION=$(go list -m -f '{{ .Version }}' ${KUBERNETES_CONFIGURATION_CRDS_PACKAGE})
 KUBERNETES_CONFIGURATION_CRDS_CRDS_LOCAL_PATH="$(go env GOPATH)/pkg/mod/${KUBERNETES_CONFIGURATION_CRDS_PACKAGE}@${KUBERNETES_CONFIGURATION_CRDS_VERSION}"
 
+# Darwin's cp doesn't support --no-preserve flag so we need to resort to cp from GNU utils / coreutils.
+# NOTE: https://apple.stackexchange.com/questions/69223/how-to-replace-mac-os-x-utilities-with-gnu-core-utilities
+CP=cp
+if [[ $(uname -s) == "Darwin" ]]; then
+  if gcp --version 2>&1 >/dev/null ; then
+    CP=gcp
```

Comment:
How about using an absolute path to the standard `homebrew` binary location (`/opt/homebrew/bin/gcp`) to avoid issue with `gcp` aliases? That might bite if folks have an alias for `gcp` set to `git cherry-pick` (which is a standard one when using `git` plugin with zsh).

Reply:
:+1 c516a09bb8650aef8f712127f1c7aa5cbe27e3e0